devtools/travis-ci/install.sh

```diff
@@ -28,7 +28,7 @@ else # Otherwise, CPython... go through conda
     if [ -z "$MINIMAL_PACKAGES" ]; then
         conda create -y -n myenv python=$PYTHON_VERSION \
             numpy scipy pandas nose openmm coverage nose-timer \
-            python-coveralls ambermini netCDF4
+            python-coveralls ambermini=16.16 netCDF4
```

Comment:
`-c omnia` won't do anything.  It'll still pull it from `ambermd` -- I tried it locally.

> Actually this is my favorite way to install package.

I presume you mean pinning versions.  I always do that now, but in the past I've always lived on the bleeding edge (since we rarely had enough people testing that).

But since moving to industry, the bleeding edge is the very last place you ever want to live (well, perhaps second-to-last after an outdated package with known security vulnerabilities).